commands.h

```diff
@@ -44,6 +44,7 @@ bool ct_scroll(arg_t);
 bool ct_drag_mark_image(arg_t);
 bool ct_select(arg_t);
 
+#ifdef _MAPPINGS_CONFIG
```

Comment:
If we want to keep _MAPPINGS_CONFIG, why not just define it here instead of guarding. It'll allow us to remove the def in main.c

Reply:
The intent here was that the keybinding macros don't get included in `commands.c` since it's not used there.